src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_archive_utils.py

```diff
@@ -0,0 +1,52 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import re
+import unittest
+
+from azure.cli.command_modules.acr._archive_utils import IgnoreRule
+
+class AcrArchiveUtilsTests(unittest.TestCase):
+    def test_ignore_rule(self):
+        test_data = (
+            ("**/bin", "user/roundrobin", False),
```

Comment:
Can you add a test to verify 

**/bin, folder1/folder2/folder3/bin, True

Reply:
updated